tests/.env

```diff
@@ -1,4 +1,3 @@
-NOSTO_SERVER_URL=localhost:3000
```

Comment:
Not needed for tests, I believe?

Reply:
Actually, I am not sure. It might be used within the Apiary service.

Reply:
I can test after merging to master and revert this change if tests fail.